README.md

```diff
@@ -1,3 +1,64 @@
+<img width="1545" alt="nr-hearts-otel" src="https://github.com/user-attachments/assets/96fb9c0e-3cc3-4319-9025-fa4a6fc48f0f">
+
+## New Relic’s Fork of the OpenTelemetry Astronomy Shop
+
+Welcome to New Relic’s fork of the [OpenTelemetry Astronomy Shop](https://opentelemetry.io/ecosystem/demo/)!
+This app is a microservice-based distributed system intended to illustrate the
+implementation of OpenTelemetry in a near real-world environment. To view the
+original repo README, scroll down or click [this link](https://github.com/newrelic/opentelemetry-demo/tree/main?tab=readme-ov-file#-opentelemetry-demo).
+
+After you follow our [quick start instructions](https://github.com/newrelic/opentelemetry-demo?tab=readme-ov-file#quick-start-with-new-relic) 
+to deploy the app, check out [how to navigate the OTLP data in your New Relic account](https://github.com/newrelic/opentelemetry-demo?tab=readme-ov-file#navigate-your-otlp-data-in-new-relic)! 
+
+### Modifications
+Please note the following modifications to our fork:
+* The helm values file has been modified to avoid breaking upstream changes
```

Comment:
@reese-lee @lazyplatypus we actually need to make a small modification to the K8s install instructions to install a specific version of the OTel demo chart.  This should prevent breaking changes.  

I'm not even sure we really need to mention this.  We just need to update the install instructions.  Thoughts?

Reply:
Good point, I included it because I had copied it down in one of the meetings. No, there's no concrete reason to include it, I'll remove this line.